tests/subsys/bluetooth/rpc_gatt_service/src/main.c

```diff
@@ -0,0 +1,414 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/ztest.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/* Include the header - it will use our stub zephyr/bluetooth/gatt.h */
```

Comment:
There's no stub gatt.h, right?

Reply:
in deed there is none. it seems like the comment is a leftover from previous iteration of the test implementation. removed it.